Program.cs

```diff
@@ -1,7 +1,18 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Input: ");
+         int dec = int.Parse (Console.ReadLine ());

```

Comment:
If non-numeric value is inputted, kindly assume it as 0. or show an error message.?